src/components/MainComposer/MainComposer.module.scss

```diff
@@ -11,49 +13,68 @@
 
   background-color: global.$secondaryPurple;
   border-radius: 2rem 2rem 0 0;
-}
 
-.innerHeader .mainContent {
-  .mainText {
-    color: global.$primaryWhite;
-    font-size: 2.2rem;
-    font-weight: 400;
-    font-style: normal;
-    line-height: 2.8rem;
-  }
+  .mainContent {
+    .mainText {
+      color: global.$primaryWhite;
+      font-size: 2.2rem;
+      font-weight: 400;
+      font-style: normal;
+      line-height: 2.8rem;
 
-  .mainText::before {
-    width: 1.5rem;
-    height: 1.5rem;
+      &::before {
+        width: 1.5rem;
+        height: 1.5rem;
 
-    display: inline-block;
+        display: inline-block;
 
-    float: left;
+        float: left;
```

Comment:
Acho que quando usar float-left nao precisa usar inlineblock, alias, float é uma propriedade um pouco desatualizada. tenta usar flexbox ou grid.

Reply:
https://dev.to/ethanmgustafson/css-float-flexbox-14fghttps://dev.to/ethanmgustafson/css-float-flexbox-14fg